src/Nethermind/Nethermind.Blockchain.Test/TxPools/Collections/DistinctValueSortedPoolTests.cs

```diff
@@ -87,7 +92,12 @@ public void Distinct_transactions_are_all_added(Transaction[] transactions, int
         [TestCase(false)]
         public void Same_transactions_are_all_replaced_with_highest_gas_price(bool gasPriceAscending)
         {
-            var pool = new DistinctValueSortedPool<Keccak, Transaction>(Capacity, (t1, t2) => t1.GasPrice.CompareTo(t2.GasPrice), PendingTransactionComparer.Default);
+            var pool = new DistinctValueSortedPool<Keccak, Transaction, Address>(
```

Comment:
Complexity of this is a bit crazy.

Reply:
simplified by reusing TxPool way of creating DistinctValueSortedPool